src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionUtilityToolFilter.java

```diff
@@ -130,10 +132,11 @@ public void doFilter(ServletRequest request, ServletResponse response,
 			chain.doFilter(useRequest, response);
 		}
 		else if(session.getAttribute(LAUNCH_TYPE).equals("sc")){
-
-			if(!checkForAuthorization(useRequest)) {
-				useResponse.sendError(403);
-				return;
+			if(!useRequest.getPathInfo().equals(STATUS_PAGE)  && !useRequest.getPathInfo().equals(PING_PAGE)){
```

Comment:
Check that the URL starts with /status rather than check for exact URLs.  The problem with using exact URLS is that it makes it more work to add new urls with little payoff for that work.